plugins/CMakeLists.txt

```diff
@@ -30,6 +31,9 @@ endif (WITH_history)
 if (WITH_history2)
   add_subdirectory( history2 )
 endif (WITH_history2)
+if (WITH_history3)
```

Comment:
‎Ok. On it.                                                                                                                                                                                                                From: KaushikSent: Sunday, 19 July 2015 11:17To: josh-wambua/kopeteReply To: josh-wambua/kopeteCc: Joseph W. JoshuaSubject: Re: [kopete] Database helper (#2)In plugins/CMakeLists.txt:
> @@ -30,6 +31,9 @@ endif (WITH_history)
>  if (WITH_history2)
>    add_subdirectory( history2 )
>  endif (WITH_history2)
> +if (WITH_history3)

Can you remove history and history2 and rename history3 as history ?

—Reply to this email directly or view it on GitHub.